public/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "POOPS Volunteer Tracker",
+  "short_name": "POOPS Volunteer Tracker",
+  "icons": [
+    {
+      "src": "/poops192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+
+    {
+      "src": "/poops512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    },
+
+    {
+      "src": "maskable.png",
+      "sizes": "96x96",
+      "type": "image/x-icon",
+      "purpose": "maskable"
+    }
+  ],
+  "theme_color": "#FFFFFF",
+  "background_color": "#FFFFFF",
+  "start_url": "/",
+  "display": "standalone",
+  "orientation": "portrait"
+}
```

Comment:
This is not needed as app.webmanifest already exist. Edge recommends it over manifest.json. 
https://web.dev/learn/pwa/web-app-manifest/